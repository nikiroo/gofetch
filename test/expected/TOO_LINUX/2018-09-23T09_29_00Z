            LINAGORA A FÊTÉ SES 18 ANS CET ÉTÉ : LA VIDÉO          

  2018/09/23

  o Reference: 2018-09-23T09_29_00Z
  o News link: https://www.toolinux.com/?Linagora-a-fete-ses-18-ans-cet-ete-la-video
  o Source link: 


    Linagora a fêté ses 18 ans cet été : la vidéo
    
    dimanche 23 septembre 2018
    
    Cet  été,  tout comme Toolinux.com, LINAGORA a soufflé sa 18ème
    bougie. Une vidéo a été publiée pour fêter cet événement.
    
    Ce  18e  anniversaire  a été l’occasion pour le groupe - devenu
    international  !  -  de  publier  une  refonte  de son site web
    autour de ses valeurs consolidées.
    
    Note  aux  lecteurs  : Linagora finance toolinux.com depuis mai
    2000.
    
    [1]
    
    
    
    [1] https://www.toolinux.com/?Linagora-a-fete-ses-18-ans-cet-e-
    te-la-video#forum



