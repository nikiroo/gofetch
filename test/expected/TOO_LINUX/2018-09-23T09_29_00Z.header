0Linagora a fêté ses 18 ans cet été : la vidéo	null/TOO_LINUX/2018-09-23T09_29_00Z		70
i  2018/09/23
i
i    Cet  été,  tout comme Toolinux.com, LINAGORA a soufflé sa 18ème
i    bougie.  Une  vidéo  a été publiée pour fêter cet événement. Ce
i    18e  anniversaire  a  été  l’occasion  pour  le groupe - devenu
i    international  !  -  de  publier  une  refonte  de son site web
i    autour   de  ses  valeurs  consolidées.  Note  aux  lecteurs  :
i    Linagora finance toolinux.com depuis mai (...)
i
