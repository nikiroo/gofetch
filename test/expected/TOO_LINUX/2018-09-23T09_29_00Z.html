<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>Linagora a fêté ses 18 ans cet été : la vidéo</h2>
	<div class='details'>(2018/09/23)</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>2018-09-23T09_29_00Z</a></li>
        <li>News link: <a href='https://www.toolinux.com/?Linagora-a-fete-ses-18-ans-cet-ete-la-video'>https://www.toolinux.com/?Linagora-a-fete-ses-18-ans-cet-ete-la-video</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Linagora a fêté ses 18 ans cet été : la vidéo<br/><br/>dimanche 23 septembre 2018<br/><br/>Cet été, tout comme Toolinux.com, LINAGORA a soufflé sa 18ème bougie. Une vidéo a été publiée pour fêter cet événement.<br/><br/>Ce 18e anniversaire a été l’occasion pour le groupe - devenu international ! - de publier une refonte de son site web autour de ses valeurs consolidées.<br/><br/>Note aux lecteurs : Linagora finance toolinux.com depuis mai 2000.<br/><br/>[1]<br/><br/><br/><br/>[1] https://www.toolinux.com/?Linagora-a-fete-ses-18-ans-cet-ete-la-video#forum
	</div>
<hr/>
</div>
</body>
